ArjunaJTS/standalone/pom.xml

```diff
@@ -187,23 +178,25 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-antrun-plugin</artifactId>
-             <version>1.6</version>
-             <executions>
-               <execution>
-                 <phase>integration-test</phase>
-                 <configuration>
-                   <target>
-                     <exec dir="${project.basedir}" executable="cmd.exe" failonerror="true">
-                       <arg value="/c"/>
-                       <arg value="run.bat"/>
-                       <env key="QUICKSTART_NARAYANA_VERSION" value="${version.narayana}" />
-                     </exec>
-                   </target>
-                 </configuration>
-                 <goals>
-                   <goal>run</goal>
-                 </goals>
-               </execution>
+            <version>1.6</version>
+            <executions>
+              <execution>
+                <phase>integration-test</phase>
+                <configuration>
+                  <target>
+                    <exec dir="${project.basedir}"
```

Comment:
I think the changes prior to this line and from line 193 are just formatting and could be reverted

Reply:
I think actually the whole of this section might be a reformat?